pkgs/tools/networking/unbound/python.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchurl, openssl, expat, libevent, swig, python, pythonPackages }:
```

Comment:
python is an attribute of pythonPackages, so no need to add both here. Having only pythonPackages will make it easier and less error-prone when using a custom python.